apps/nowcasting-app/components/map/pvLatest.tsx

```diff
@@ -41,7 +41,7 @@ const useGetForecastsData = (isNormalized: boolean) => {
     axiosFetcherAuth,
     {
       onSuccess: (data) => {
-        setForecastCreationTime(data.forecasts[0].forecastCreationTime);
+        if (data.forecasts?.length) setForecastCreationTime(data.forecasts[0].forecastCreationTime);
```

Comment:
just asking out of curiosity why did this `if` statement get added?

Reply:
As part of the stuff not loading properly, this was causing an error if the data was coming back empty, so just added an extra check. Not perfect still by any means still!